### PR TITLE
Use port 8081 for nexus health check

### LINKED ISSUE #10: Use port for health check
As Nexus returns a `500 Server Error` for a request to http://localhost:8081/nexus/service/metrics/healthcheck if one of their health checks fails, we should check if Port 8081 is open instead.

### PR DESCRIPTION
**Pull request by robertauer**
Resolves #10

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -81,10 +81,7 @@ func env(key string) string {
 }
 
 func waitUntilNexusBecomesReady(url string, username string, password string) error {
```
Comment (ppxl) [inline app.go]: Unused parameters:
All of the given parameters are no longer used. Shouldn't these be removed as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Use port 8081 for nexus health check; #10
```
Comment (ppxl) [inline CHANGELOG.md]: I'd like to see a little background explanation about the reason of the change. I, too, consider the nexus behaviour to return the health JSON along with an HTTP 500 as really bad practice. At least other developers should know about this.

An alternative could be a comment in the git commit message of the actual change but that was empty.

What do you think?

Comment (robertauer) [reply CHANGELOG.md]: The issue #10 says a lot about the problem. I think the changelog should only summarize what has changed and if you want to know more, you can read the issue.

File: app.go (hunk 1 above)
Comment (robertauer) [reply app.go]: Yes, they should. I'll fix that.

File: CHANGELOG.md (hunk 2 above)
Comment (ppxl) [reply CHANGELOG.md]: Hmm, okay, that totally counts. Thanks

**Review by ppxl — APPROVED** [review]
LGTM

**Pull request merged**